.mill-version

```diff
@@ -0,0 +1 @@
+0.11.7
```

Comment:
This file is not required, the rocket-chip use nix to handle the version. it will introduce downloading artifacts from the internet.